Container App/src/Components/Header/Header.js

```diff
@@ -1,28 +1,135 @@
-import React from 'react';
-import { NavLink } from 'react-router-dom';
-import './Header.css';
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import {
+  BottomNavigation,
+  BottomNavigationAction,
+  makeStyles,
+  Select,
+  MenuItem,
+  InputLabel,
+  FormControl,
+} from "@material-ui/core";
+import Computer from "@material-ui/icons/Computer";
+import FeaturedPlayListIcon from "@material-ui/icons/FeaturedPlayList";
+import AccountCircle from "@material-ui/icons/AccountCircle";
+import CreditCard from "@material-ui/icons/CreditCard";
+import AccountBalance from "@material-ui/icons/AccountBalance";
+import helixLogo from "./helixLogo.jpg";
 
 //TODO Make Company List Dynamic
 
-const Header = () => (
-  <header>
-    <div className="center-column">
-      <h1>Put New Header Here From Dashboard Component</h1>
-    </div>
-    <nav>
-      <ol className="center-column">
-        <li>
-          <NavLink to="/dashboard">Dashboard</NavLink>
-        </li>
-        <li>
-          <select name="companyList" id="companies">
-            <option value="Microsoft">Microsoft</option>
-            <option value="CNB">City National Bank</option>
-          </select>       
-        </li>
-      </ol>
-    </nav>
-  </header>
-);
+const useStyles = makeStyles((theme) => ({
```

Comment:
Lets find some time tomorrow to go over this. I want to see what we can pull out into a theme and have that as a separate file.